packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -72,7 +72,7 @@ const SubscriptionUtils = (module.exports = {
 async function fetchSubscribedProductsFromStripe(uid, stripeHelper, email) {
   const customer = await stripeHelper.customer(uid, email);
   if (!customer || !customer.subscriptions.data) {
-    return;
+    return [];
```

Comment:
The `undefined` returned from here could overwrite the default set at https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/utils/subscriptions.js#L38, causing errors when later code call array functions on it (`includes`, `reduce`, whathaveyou).